### PR TITLE
home task_04

### PR DESCRIPTION
**Pull request by antonakulich1993**

### CODE REVIEW

File: aakulich/aakulich3.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+     <p><img title="Python" src="E:\programm\Icon-Python.png" align="right" alt="PythonSymbol"></p>
+     <ul>
+          <li><a href="aakulich2.html">Программа на Python — часть 1</a></li>
+          <li><a href="aakulich3.html">Программа на Python — часть 2</a></li>
+          <li><a href="aakulich4.html">Программа на Python — часть 3</a></li>
+     </ul>
+     <h2><a name="содержание">Содержание</h2></a>
+     <h3>>HOW-TO: Программа на Python, Часть 2</h3>
+     <ul>
+       <a href="#списки"><li>Списки</li></a>
```
Comment (pluhin) [inline aakulich/aakulich3.html:24]: Why tag a outside li?

File: aakulich/aakulich4.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  <p><img title="Python" src="E:\programm\Icon-Python.png" align="right" alt="PythonSymbol"></p>
+  <ul>
+     <li><a href="aakulich2.html">Программа на Python — часть 1</a></li>
+     <li><a href="aakulich3.html">Программа на Python — часть 2</a></li>
+     <li><a href="aakulich4.html">Программа на Python — часть 3</a></li>
+  </ul>
+  <h2><a name="содержание">Содержание</h2></a>
+  <h3>HOW-TO: Программа на Python, Часть 3</h3>
+  <ul>
+    <li><a href="#модули"></a>Модули</li>
```
Comment (pluhin) [inline aakulich/aakulich4.html:27]: You missed text for links

File: aakulich/aakulichbirthday.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<HTML>
+    <HEAD>
+            <META charset="utf-8">
+            <title>Месяц моего дня рождения</title>
+    </HEAD>
+    <BODY>
+        <TABLE border="2" width="360"; align="center">
+        <CAPTION>Месяц моего дня рождения</CAPTION>
+        <col width= "60"><col width= "60"><col width= "60"><col width= "60"><col width= "60"><col width= "60">
```
Comment (pluhin) [inline aakulich/aakulichbirthday.html]: You should not put whitespace after =, cold you please remove that

**Review by pluhin — CHANGES_REQUESTED** [review]

**Review by antonakulich1993 — COMMENTED** [review]
Watch css task

File: aakulich/aakulich1.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+         <h2>Тег address</h2>
+         <adress><i><u>Kuprevicha, 1/1, 220141, Minsk, Belarus. Phone: +375.17.2686911</u></i></adress>
+         <hr>
+         <h2>Семантические и функциональные строчные теги</h2>
+         <p>Пожелания автора:</p>
+         <s> Данный текст убрать из монографии</s> и заменить его на <u>вновь помещённый текст.</u>
+         <hr>
+         <h2>Работа со строчными и блоковыми цитатами</h2>
+         <p>Цитата короткая:</p>
+         <blockquote>И всё­таки она вертится!</blockquote>
```
Comment (pluhin) [inline aakulich/aakulich1.html]: here need short quote

File: aakulich/aakulich1.html [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+         <p>Цитата занимающая несколько предложений:</p>
+                     <pre>О сколько нам открытий чудных
```
Comment (pluhin) [inline aakulich/aakulich1.html]: and here - blockquote

File: aakulich/aakulich2.html [hunk 6]
```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<HTML>
+    <HEAD>>
+        <META charset="utf-8">
+        <TITLE>Программа на Python</TITLE>
+    </HEAD>
+    <BODY>
+         <h1>HOW-TO: Программа на Python, Часть 1</h1>
+         <p><img title="Python" src="E:\programm\Icon-Python.png" align="right" alt="PythonSymbol"></p>
```
Comment (pluhin) [inline aakulich/aakulich2.html]: Need to use relative path!!! There is no disk E on github!!!

File: aakulich/aakulich2.html [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+        <p><a href=#содержание>К содежанию</a></p>
+        <p>Теперь давайте рассмотрим пример программирования цикла. Запустите текстовый
+        редактор и наберите следующую программу:
+<pre>#!/usr/bin/env python
+for cntr in range(0,10):
+print cntr</pre>
+        <i>Не забудьте поставить символ табуляции перед строкой «print cntr».</i> Это важно. В
+        Python для указания на блоки кода используются не круглые «(» и фигурные «{» скобки
+        как в других языках программирования, а отступы.</p>
+        <a href="aakulich2.html"><img align="left" title="previous" src="E:\programm\prev.png" alt="previous"></a>
```
Comment (pluhin) [inline aakulich/aakulich2.html]: Need to use relative path!!!

File: aakulich/aakulich3.html [hunk 8]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<HTML>
+    <HEAD>>
+        <META charset="utf-8">
+        <TITLE>Программа на Python</TITLE>
+    </HEAD>
+    <BODY>
+     <h1>HOW-TO: Программа на Python, Часть 2</h1>
+     <p><img title="Python" src="E:\programm\Icon-Python.png" align="right" alt="PythonSymbol"></p>
```
Comment (pluhin) [inline aakulich/aakulich3.html]: Need to use relative path!!!

File: aakulich/aakulich2.html [hunk 9]
```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<HTML>
+    <HEAD>>
```
Comment (pluhin) [inline aakulich/aakulich2.html]: Double >>

File: aakulich/aakulich1.html [hunk 10]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<HTML>
+    <HEAD>>
```
Comment (pluhin) [inline aakulich/aakulich1.html]: Double >>

File: aakulich/aakulich3.html [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+     <ul>
+       <a href="#списки"><li>Списки</li></a>
+       <a href="#комментарии"><li>Комментарии</li></a>
+     </ul>
+     <hr>     
+     <p>Впрошлом уроке мы рассмотрели простую программу, которая использовала raw_input
+      для получения ответа пользователя, простые типы переменных и простой цикл «<b>for</b>». В
+      этом уроке мы рассмотрим ещё больше типов переменных и напишем ещё больше
+      программ.</p>
+     <a name="списки"><h2>Списки</h2></a>
```
Comment (pluhin) [inline aakulich/aakulich3.html]: tag a should be inside header

Comment (pluhin) [inline aakulich/aakulich3.html]: And do not use Cyrillic  for name and id attributes

File: aakulich/aakulich4.html [hunk 12]
```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<HTML>
+ <HEAD>
+    <META charset="utf-8">
+    <TITLE>Программирование на Python</TITLE>
+ </HEAD>
+ <BODY>
+  <h1>HOW-TO: Программа на Python, Часть 3</h1>
+  <p><img title="Python" src="E:\programm\Icon-Python.png" align="right" alt="PythonSymbol"></p>
```
Comment (pluhin) [inline aakulich/aakulich4.html]: Need to use relative path!!!

File: aakulich/aakulich4.html [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+      print '%d - %d = %d ' % (num1,num2,num1-num2)
+print '\n'
+DoTwo(1,2)
+DoTwo(1,4)
+DoTwo(10,5)</pre>
+        
+  <p>Как видите, здесь написано меньше кода &#8212 8 строк вместо 12. Если нам нужно будет что-
+  либо поменять в функции, мы сможем сделать это, не создавая проблем для нашей
+  программы. Чтобы вызвать функцию, нужно написать её имя и параметры.</p>
+       <a href="aakulich3.html"><img align="left" title="previous" src="E:\programm\prev.png" alt="previous"></a>
```
Comment (pluhin) [inline aakulich/aakulich4.html]: Need to use relative path!!!

File: aakulich/forms.html [hunk 14]
```diff
@@ -0,0 +1,135 @@
...
+        <h3>Простая форма поиска с вариантами:</h3>
+        <form method="GET" action="http://sc.it-academy.by/fe/form.php">
+        <fieldset>
+        <legend>Строка поиска</legend>
+        <input type="text" id="login" name="search" autofocus>
+        <input type="submit" value="искать">
+        <label><br>
+        <input type="radio" name="search_dist" value="site">на сайте
+        <input type="radio" name="search_dist" value="google" checked>в google<br>
+        </form>
```
Comment (pluhin) [inline aakulich/forms.html:34]: wrong sequence of closing tags

File: aakulich/forms.html [hunk 15]
```diff
@@ -0,0 +1,135 @@
...
+        </td>
+        </fieldset>
+        <td> 
+        <h3>Простая форма поиска с вариантами:</h3>
+        <form method="GET" action="http://sc.it-academy.by/fe/form.php">
+        <fieldset>
+        <legend>Строка поиска</legend>
+        <input type="text" id="login" name="search" autofocus>
+        <input type="submit" value="искать">
+        <label><br>
```
Comment (pluhin) [inline aakulich/forms.html:31]: here should be two labels

File: aakulich/forms.html [hunk 16]
```diff
@@ -0,0 +1,135 @@
...
+        </form>
+        </label>
+        </td>
+        </fieldset>
+        <td> 
+        <h3>Простая форма поиска с вариантами:</h3>
+        <form method="GET" action="http://sc.it-academy.by/fe/form.php">
+        <fieldset>
+        <legend>Строка поиска</legend>
+        <input type="text" id="login" name="search" autofocus>
```
Comment (pluhin) [inline aakulich/forms.html:29]: Not all requirements for this field have been implemented

File: aakulich/forms.html [hunk 17]
```diff
@@ -0,0 +1,135 @@
...
+        <td>    
+        <h3>Простая форма авторизации</h3>
+        <form method="post" action="http://sc.it-academy.by/fe/form.php">
+        <fieldset>
+        <legend>авторизируйтесь</legend>
+        <input type="text" id="login" name="login"required> 
+        <label for="login">*Логин</label><br>
+        <input type="password" id="password" name="password" required>
+        <label for="password">*Пароль</label><br>
+        <input type="checkbox" name="check">Оставаться в сети
```
Comment (pluhin) [inline aakulich/forms.html]: where is label for checksbox?

File: aakulich/forms.html [hunk 18]
```diff
@@ -0,0 +1,135 @@
...
+        <fieldset>  
+        <legend>Как вам?</legend>
+        <h3>Опрос(голосование):</h3>
+        <form method="GET" action="http://sc.it-academy.by/fe/form.php">
+        <input type="radio" name="search_dist" value="ничего">ничего<br>
+        <input type="radio" name="search_dist" value="неплохо">неплохо<br>
+        <input type="radio" name="search_dist" value="хорошо">хорошо<br>
+        <input type="radio" name="search_dist" value="супер">супер<br>
+        <input type="submit" value="отправить"><br>
+        <input type="hidden" name=date_voite, value="2013.12.31">
```
Comment (pluhin) [inline aakulich/forms.html]: why is comma doing here?

File: aakulich/forms.html [hunk 19]
```diff
@@ -0,0 +1,135 @@
...
+        </fieldset>      
+        </td>    
+        </form>
```
Comment (pluhin) [inline aakulich/forms.html]: close part of form tag shouldn't be here

File: aakulich/forms.html [hunk 20]
```diff
@@ -0,0 +1,135 @@
...
+        </tr>
+        <tr>
+        <td>
+        <h3>Отзыв:</h3>
+        <form method="POST" action="http://sc.it-academy.by/fe/form.php">
+        <fieldset>
+        <legend>Оставьте ваш отзыв:</legend>
+        <input type="text" id="login" placeholder required name="login" >*Имя<br>
```
Comment (pluhin) [inline aakulich/forms.html]: Where is label here?

File: aakulich/forms.html [hunk 21]
```diff
@@ -0,0 +1,135 @@
...
+        <input type="email" name="email" placeholder="это обязательно поле" required>*e-mail<br>
```
Comment (pluhin) [inline aakulich/forms.html]: Where is label here?

File: aakulich/forms.html [hunk 22]
```diff
@@ -0,0 +1,135 @@
...
+        <textarea name="форма отправки"></textarea>
+        <input type="submit" value="отправить"><br>
+        </fieldset>      
+        </form>
+        </td>
+       
+        <td>
+        <h3>Выпадающий и мульти списки:</h3>
+        <form method="POST" action="http://sc.it-academy.by/fe/form.php">
```
Comment (pluhin) [inline aakulich/forms.html:86]: Just one list here

File: aakulich/forms.html [hunk 23]
```diff
@@ -0,0 +1,135 @@
...
+        </tr>
+        <tr>
+        <td>
+        <h3>Загрузка файлов на сервер:</h3>
+        <form method="POST" enctype="multipart/form-data" action="http://sc.it-academy.by/fe/form.php">
+        <fieldset>
+        <legend>Upload</legend>
+        <legend>Файл:</legend>
+        <input type="file" name="file_field" dir="ltr"><br>
+        <legend>Срок хранения (года):</legend>
```
Comment (pluhin) [inline aakulich/forms.html]: why legend? label and for also here

Comment (pluhin) [inline aakulich/forms.html]: And how are you going to send how many years?

File: aakulich/forms.html [hunk 24]
```diff
@@ -0,0 +1,135 @@
...
+        </td>
+        <td>
+        <h3>Валидация данных HTML5:</h3>
+        <form method="POST" action="http://sc.it-academy.by/fe/form.php">
+        <fieldset>
+        <legend>Строка поиска:</legend>
+        <input type="text" placeholder="только русские буквы" lang="ru"><br>
+        <legen>Телефонный норме в формате</legend>
+        <legend>+ХХХ-ХХ-ХХХХХХХ, где Х - любое числоы</legend>
+        <input type="text" placeholder="+375-29-1234567" pattern="+-[0,9]{3}-[0,9]{2}-[0,9]{7}"><br>
```
Comment (pluhin) [inline aakulich/forms.html]: wrong pattern here

File: aakulich/forms.html [hunk 25]
```diff
@@ -0,0 +1,135 @@
...
+        </form>
+        </td>
+        <td>
+        <h3>Валидация данных HTML5:</h3>
+        <form method="POST" action="http://sc.it-academy.by/fe/form.php">
+        <fieldset>
+        <legend>Строка поиска:</legend>
+        <input type="text" placeholder="только русские буквы" lang="ru"><br>
+        <legen>Телефонный норме в формате</legend>
+        <legend>+ХХХ-ХХ-ХХХХХХХ, где Х - любое числоы</legend>
```
Comment (pluhin) [inline aakulich/forms.html]: Why legend? Label here

File: aakulich/forms.html [hunk 26]
```diff
@@ -0,0 +1,135 @@
...
+        <input type="submit" dir="rtl"><br>
+        </fieldset>
+        </form>
+        </td>
+        <td>
+        <h3>Валидация данных HTML5:</h3>
+        <form method="POST" action="http://sc.it-academy.by/fe/form.php">
+        <fieldset>
+        <legend>Строка поиска:</legend>
+        <input type="text" placeholder="только русские буквы" lang="ru"><br>
```
Comment (pluhin) [inline aakulich/forms.html]: Where is pattern?

**Review by pluhin — CHANGES_REQUESTED** [review]

File: aakulich/aakulich4.html (hunk 2 above)
Comment (antonakulich1993) [reply aakulich/aakulich4.html:27]: i make some correction of mistakes , i also have a question . how can i make clone of my git hub home computer , to work computer

Comment (pluhin) [reply aakulich/aakulich4.html:27]: > i make some correction of mistakes , i also have a question . how can i make clone of my git hub home computer , to work computer

https://sites.google.com/site/feexpr/Downhome/git
steps start from section
"Устанавливаем клиент git. Вы можете установить любой другой клиент. Устанавливается клиент со всеми настройками по умолчанию....."

**Comment by pluhin** [comment]
1. You didn't correct all my comments
2. You have conflicts

File: aakulich/aakulich3.html (hunk 1 above)
Comment (antonakulich1993) [reply aakulich/aakulich3.html:24]: i did it , but i dont have my files on the other computer

File: aakulich/forms.html [hunk 27]
```diff
@@ -61,27 +61,28 @@ <h3>Опрос(голосование):</h3>
...
         </tr>
         <tr>
         <td>
         <h3>Отзыв:</h3>
         <form method="POST" action="http://sc.it-academy.by/fe/form.php">
         <fieldset>
         <legend>Оставьте ваш отзыв:</legend>
-        <input type="text" id="login" placeholder required name="login" >*Имя<br>
-        <input type="email" name="email" placeholder="это обязательно поле" required>*e-mail<br>
+        <input type="text" id="login" placeholder required name="login" ><label>*Имя</label><br>
```
Comment (pluhin) [inline aakulich/forms.html:76]: label  for=?

File: aakulich/forms.html [hunk 28]
```diff
@@ -61,27 +61,28 @@ <h3>Опрос(голосование):</h3>
...
+        <input type="email" name="email" placeholder="это обязательно поле" required><label>*e-mail</label><br>
```
Comment (pluhin) [inline aakulich/forms.html:77]: label for=

**Review by pluhin — CHANGES_REQUESTED** [review]

File: aakulich/aakulich2.html [hunk 29]
```diff
@@ -59,10 +61,12 @@ <h2><a name="пример"></a>Пример цикла For</h2>
...
         Python для указания на блоки кода используются не круглые «(» и фигурные «{» скобки
         как в других языках программирования, а отступы.</p>
-        <a href="aakulich2.html"><img align="left" title="previous" src="E:\programm\prev.png" alt="previous"></a>
-        <a href="aakulich3.html"><img align="right" title="next" src="E:\programm\next.png" alt="next"></a>
+
+        <a href="aakulich2.html"><img align="left" title="previous" src=" prev.png" alt="previous"></a>
+        <a href="aakulich3.html"><img align="right" title="next" src="next.png" alt="next"></a>
         <div align="center">
-        <a href="#"><img title="home" src="E:\programm\home.png" alt="home"></a>
+        <a href="#"><img title="home" src="E:home.png" alt="home"></a>
```
Comment (pluhin) [inline aakulich/aakulich2.html:68]: E: - ??

File: aakulich/aakulich3.html [hunk 30]
```diff
@@ -15,8 +17,13 @@ <h1>HOW-TO: Программа на Python, Часть 2</h1>
      <h2><a name="содержание">Содержание</h2></a>
      <h3>>HOW-TO: Программа на Python, Часть 2</h3>
      <ul>
+<<<<<<< HEAD
```
Comment (pluhin) [inline aakulich/aakulich3.html:20]: need to remove this line

File: aakulich/aakulich3.html [hunk 31]
```diff
@@ -15,8 +17,13 @@ <h1>HOW-TO: Программа на Python, Часть 2</h1>
...
+       <li></li><a href="#списки">Списки</a></li>
+       <li></li><a href="#комментарии">Комментарии</a></li>
+=======
```
Comment (pluhin) [inline aakulich/aakulich3.html:23]: need to remove this line

File: aakulich/aakulich3.html [hunk 32]
```diff
@@ -15,8 +17,13 @@ <h1>HOW-TO: Программа на Python, Часть 2</h1>
...
        <a href="#списки"><li>Списки</li></a>
        <a href="#комментарии"><li>Комментарии</li></a>
+>>>>>>> my/m-sc1-59-19
```
Comment (pluhin) [inline aakulich/aakulich3.html:26]: need to remove this line

File: aakulich/aakulich3.html [hunk 33]
```diff
@@ -15,8 +17,13 @@ <h1>HOW-TO: Программа на Python, Часть 2</h1>
      <h2><a name="содержание">Содержание</h2></a>
      <h3>>HOW-TO: Программа на Python, Часть 2</h3>
      <ul>
+<<<<<<< HEAD
+       <li></li><a href="#списки">Списки</a></li>
+       <li></li><a href="#комментарии">Комментарии</a></li>
+=======
        <a href="#списки"><li>Списки</li></a>
        <a href="#комментарии"><li>Комментарии</li></a>
```
Comment (pluhin) [inline aakulich/aakulich3.html:25]: tag a should be inside li

File: aakulich/aakulich3.html [hunk 34]
```diff
@@ -15,8 +17,13 @@ <h1>HOW-TO: Программа на Python, Часть 2</h1>
      <h2><a name="содержание">Содержание</h2></a>
      <h3>>HOW-TO: Программа на Python, Часть 2</h3>
      <ul>
+<<<<<<< HEAD
+       <li></li><a href="#списки">Списки</a></li>
+       <li></li><a href="#комментарии">Комментарии</a></li>
+=======
        <a href="#списки"><li>Списки</li></a>
```
Comment (pluhin) [inline aakulich/aakulich3.html:24]: tag a should be inside li

File: aakulich/aakulich4.html [hunk 35]
```diff
@@ -6,17 +6,27 @@
...
-  <p><img title="Python" src="E:\programm\Icon-Python.png" align="right" alt="PythonSymbol"></p>
+
+  <p><img title="Python" src="Icon-Python.png" align="right" alt="PythonSymbol"></p>
+
   <ul>
      <li><a href="aakulich2.html">Программа на Python — часть 1</a></li>
      <li><a href="aakulich3.html">Программа на Python — часть 2</a></li>
      <li><a href="aakulich4.html">Программа на Python — часть 3</a></li>
   </ul>
+<<<<<<< HEAD
```
Comment (pluhin) [inline aakulich/aakulich4.html:17]: conflict section here, resolve please

File: aakulich/aakulichbirthday.html [hunk 36]
```diff
@@ -7,7 +7,9 @@
     <BODY>
         <TABLE border="2" width="360"; align="center">
```
Comment (pluhin) [inline aakulich/aakulichbirthday.html:8]: why ; here?

File: aakulich/forms.html [hunk 37]
```diff
@@ -0,0 +1,136 @@
...
+        <tr>
+        <td>
+        <h3>Простоя форма поиска с вариантами:</h3>
+        <form method="GET" action="http://sc.it-academy.by/fe/form.php">
+        <fieldset>
+        <legend>Строка поиска</legend>
+        <input type="text" id="login" name="search" autofocus maxlength="10">
+        <input type="submit" value="искать">
+        <br>
+        <input type="radio" name="search_dist" value="site"><label>на сайте</label>
```
Comment (pluhin) [inline aakulich/forms.html:18]: wrong construction for the label

File: aakulich/forms.html [hunk 38]
```diff
@@ -0,0 +1,136 @@
...
+        <input type="radio" name="search_dist" checked value="google"><label>в google</label><br>
```
Comment (pluhin) [inline aakulich/forms.html:19]: wrong construction for the label

File: aakulich/forms.html [hunk 39]
```diff
@@ -0,0 +1,136 @@
...
+        </form>
+        </label>
```
Comment (pluhin) [inline aakulich/forms.html:21]: What is the close part doing here?

File: aakulich/forms.html [hunk 40]
```diff
@@ -0,0 +1,136 @@
...
+        </fieldset>
+        <td> 
+        <h3>Простая форма поиска с вариантами:</h3>
+        <form method="GET" action="http://sc.it-academy.by/fe/form.php">
+        <fieldset>
+        <legend>Строка поиска</legend>
+        <input type="text" id="login" name="search" autofocus>
+        <input type="submit" value="искать">
+        <label><br>
+        <input type="radio" name="search_dist" value="site">на сайте
```
Comment (pluhin) [inline aakulich/forms.html:32]: Where is label?

File: aakulich/forms.html [hunk 41]
```diff
@@ -0,0 +1,136 @@
...
+        <input type="radio" name="search_dist" value="google" checked>в google<br>
```
Comment (pluhin) [inline aakulich/forms.html:33]: Where is label

File: aakulich/forms.html [hunk 42]
```diff
@@ -0,0 +1,136 @@
...
+        <td>    
+        <h3>Простая форма авторизации</h3>
+        <form method="post" action="http://sc.it-academy.by/fe/form.php">
+        <fieldset>
+        <legend>авторизируйтесь</legend>
+        <input type="text" id="login" name="login"required> 
+        <label for="login">*Логин</label><br>
+        <input type="password" id="password" name="password" required>
+        <label for="password">*Пароль</label><br>
+        <input type="checkbox" name="check"><label>Оставаться в сети</label>
```
Comment (pluhin) [inline aakulich/forms.html:49]: wrong construction for the label

File: aakulich/forms.html [hunk 43]
```diff
@@ -0,0 +1,136 @@
...
+        <input type="submit" value="Войти">
+        </fieldset>
+        </form>
+        </td>
+        <td>
+        <fieldset>
```
Comment (pluhin) [inline aakulich/forms.html:55]: ```fieldset``` and ```legend``` should inside tag ```form```

File: aakulich/forms.html [hunk 44]
```diff
@@ -0,0 +1,136 @@
...
+            <option value="Брест">Брест</option>
+            <option value="Минск">Минск</option>          
+            <option value="Витебск">Витебск</option> 
+        </optgroup>
+        <optgroup label="Города России"> 
+            <option value="Москва">Москва</option>
+            <option value="Рязань">Рязань</option>          
+            <option value="Магадан">Магадан</option> 
+        </optgroup>
+        <input type="submit" value="OK">
```
Comment (pluhin) [inline aakulich/forms.html:100]: Whatis the input doing inside select

File: aakulich/forms.html [hunk 45]
```diff
@@ -0,0 +1,136 @@
...
+        </form>    
+        </td> 
+        </tr>
+        <tr>
+        <td>
+        <h3>Загрузка файлов на сервер:</h3>
+        <form method="POST" enctype="multipart/form-data" action="http://sc.it-academy.by/fe/form.php">
+        <fieldset>
+        <legend>Upload</legend>
+        <label>Файл:<br>
```
Comment (pluhin) [inline aakulich/forms.html:112]: label for?

File: aakulich/forms.html [hunk 46]
```diff
@@ -0,0 +1,136 @@
...
+        <input type="file" name="file_field" dir="ltr"><br>
+        Срок хранения (года):<br></label>
```
Comment (pluhin) [inline aakulich/forms.html:114]: two fields - two labels

Comment (pluhin) [inline aakulich/forms.html:114]: Where is input for sending years of storage?

File: aakulich/forms.html [hunk 47]
```diff
@@ -0,0 +1,136 @@
...
+        </fieldset>
+        </form>
+        </td>
+        <td>
+        <h3>Валидация данных HTML5:</h3>
+        <form method="POST" action="http://sc.it-academy.by/fe/form.php">
+        <fieldset>
+        <legend>Строка поиска:</legend>
+        <input type="text" placeholder="только русские буквы" lang="ru" patter="[А-Я][а-я]+"><br>
+        <label>Телефонный норме в формате<br>
```
Comment (pluhin) [inline aakulich/forms.html:125]: Label for?

File: aakulich/forms.html [hunk 48]
```diff
@@ -0,0 +1,136 @@
...
+         +ХХХ-ХХ-ХХХХХХХ, где Х - любое числоы<br></label>
+        <input type="text" placeholder="+375-29-1234567" pattern="\s{0,}\+{1,1}375\s{0,}\({0,1}(([2]{1}([5]{1}|[9]{1}))|([3]{1}[3]{1})|([4]{1}[4]{1}))\)\s{0,}[0-9]{3,3}\s{0,}[0-9]{4,4}"><br>
```
Comment (pluhin) [inline aakulich/forms.html:127]: To complicate, could you please find short solution

File: aakulich/letteraakulich.html [hunk 49]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html> 
+<HTML> 
+<HEAD><META charset="utf-8"><TITLE>Пример страницы</TITLE>
+<style>
+@import "letteraakulich.css"
```
Comment (pluhin) [inline aakulich/letteraakulich.html:5]: I cannot find this file in your PR

**Review by pluhin — CHANGES_REQUESTED** [review]

**Comment by pluhin** [comment]
Any updates?

**Pull request closed without merging**